applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -377,6 +522,23 @@ static void stream_stopped_cb(struct bt_audio_stream *stream)
 		ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
 		ERR_CHK(ret);
 	}
+
+	if (stream->user_data != NULL) {
+		struct le_audio_headset *user_headset =
+			(struct le_audio_headset *)stream->user_data;
+
+		LOG_INF("User data found in stop callback");
+
+		if (user_headset->qos_update) {
+			ret = bt_audio_stream_qos(stream->conn, unicast_group);
+			if (ret) {
+				LOG_ERR("QoS not set for stream %p: %d", stream, ret);
+			}
+			user_headset->qos_update = false;
+
+			LOG_DBG("Update QoS for stream %p: %d us", stream, stream->qos->pd);
```

Comment:
```suggestion
			LOG_DBG("Updated QoS for stream %p: %d us", stream, stream->qos->pd);
```

Reply:
Code modified and this is no longer relevant.